trunner/device.py

```diff
@@ -39,12 +40,44 @@ def wait_for_dev(port, timeout=0):
 
     # naive wait for dev
     while not os.path.exists(port):
-        time.sleep(0.5)
-        asleep += 0.5
+        time.sleep(0.01)
+        asleep += 0.01
         if timeout and asleep >= timeout:
             raise TimeoutError
 
 
+def switch_rpi_ports(state):
+    if state.lower() == 'on':
+        bool_state = 1
+    elif state.lower() == 'off':
+        bool_state = 0
+    else:
+        print('state argument should be "on" or "off"')
+        return
+    uhubctl = subprocess.run([
+        'uhubctl',
+        '-l', '2',
+        '-a', f'{bool_state}'],
+        stdout=subprocess.DEVNULL
+    )
+    if uhubctl.returncode != 0:
+        logging.error('uhubctl failed!\n')
+        raise Exception(f'RPi ports power {state} failed!')
+
+
+def unbind_rpi_usb(port_address):
```

Comment:
does unbinding USB device make the restart quicker?

Reply:
Yes, there is no need to wait for disappear. All tests take about 2 minutes.